justfile

```diff
@@ -152,12 +152,58 @@ web-dev-up:
     rm "./web-api/assets" || :
     ln -s "./assets/smbert_v0003" "./web-api/assets"
     compose="$(command -v podman-compose || command -v docker-compose)"
-    $compose -f "./web-api/compose.yml" up --detach --remove-orphans
+    $compose -f "./web-api/compose.db.yml" up --detach --remove-orphans
 
 web-dev-down:
     #!/usr/bin/env -S bash -eux -o pipefail
     compose="$(command -v podman-compose || command -v docker-compose)"
-    $compose -f "./web-api/compose.yml" down
+    $compose -f "./web-api/compose.db.yml" down
+
+build-service-image $CRATE_PATH $BIN $ASSET_DIR="":
+    #!/usr/bin/env -S bash -eux -o pipefail
+    ociBuilder="$(command -v podman || command -v docker)"
+    # if this is further nested add more .. to the soft-link below
+    out=.tmp/oci-build-dir
+    [ -d "$out" ] && rm -r "$out"
+    mkdir -p "$out"
+    cargo install \
+        --path "$CRATE_PATH" \
+        --bin "$BIN" \
+        --debug \
+        --root "$out"
+    # rename binary as to not needing to modify the Dockerfile
+    mv "$out/bin/$BIN" "$out/server.bin"
+    rmdir "$out/bin"
+    if [ -z "$ASSET_DIR" ]; then
+        ASSET_DIR="$out/assets"
+        mkdir -p "$out/assets"
+    fi
+    "$ociBuilder" build -f "$CRATE_PATH/Dockerfile" -t "$BIN" --build-arg base="$out" --build-arg assets="$ASSET_DIR"  .
```

Comment:
We can also prefix the name with `xayn` or something similar

Reply:
Sure, through docker hub has scopes like the `curlimages` in  `curlimages/curl` to make such prefixes  unnecessary.